src/bin/cargo/commands/owner.rs

```diff
@@ -6,25 +6,68 @@ use cargo::util::auth::Secret;
 pub fn cli() -> Command {
     subcommand("owner")
         .about("Manage the owners of a crate on the registry")
-        .arg_quiet()
         .arg(Arg::new("crate").action(ArgAction::Set))
-        .arg(
-            multi_opt(
-                "add",
-                "LOGIN",
-                "Name of a user or team to invite as an owner",
-            )
-            .short('a'),
-        )
-        .arg(
-            multi_opt(
-                "remove",
-                "LOGIN",
-                "Name of a user or team to remove as an owner",
-            )
-            .short('r'),
+        .arg_required_else_help(true)
+        .override_usage(
+            "\
+       cargo owner [OPTIONS] add    OWNER_NAME <CRATE_NAME>
+       cargo owner [OPTIONS] remove OWNER_NAME <CRATE_NAME>
+       cargo owner [OPTIONS] list   <CRATE_NAME>",
         )
-        .arg(flag("list", "List owners of a crate").short('l'))
+        .arg_quiet()
+        .subcommands([
+            Command::new("add")
+                .about("Name of a user or team to invite as an owner")
+                .override_usage(
+                    "\
+                   cargo owner [OPTIONS] add [OWNER_NAME] <CRATE_NAME>",
+                )
```

Comment:
Why does this need an overridden usage?

(ditto for others)

Reply:
For complete help, I used the `override_usage` method.

Before use:
```
$ cargo owner add
error: the following required arguments were not provided:
  <OWNER_NAME>

Usage: cargo owner add <OWNER_NAME> [CRATE_NAME]

For more information, try '--help'.
```

After use:
```
$ cargo owner add
error: the following required arguments were not provided:
  <OWNER_NAME>

Usage: cargo owner [OPTIONS] add <OWNER_NAME> [CRATE_NAME]

For more information, try '--help'.
```

Reply:
Looks like overridden usage has been removed, marking this as resolved